assets/in

```diff
@@ -72,7 +73,15 @@ for branch in $fetch; do
   git branch $branch FETCH_HEAD
 done
 
+return_ref=""
+if [ -n "$tag_filter" ]; then
```

Comment:
we tested the command below, $tag_filter will only match exactly, it seems like it is not taking in glob patterns.